packages/core/src/config.ts

```diff
@@ -304,7 +304,8 @@ export function composeMinifyConfig(config: LibConfig): RsbuildConfig {
         jsOptions: {
           minimizerOptions: {
             mangle: false,
-            minify: false,
+            // MF assets are loaded over the network, which means they will not be compressed by the project. Therefore, minifying them is necessary.
```

Comment:
Can we add a test case to check if the MF output is minified?

Reply:
We could add some config snapshot in unit test for 'MF', just like `esm` and `cjs` does https://github.com/web-infra-dev/rslib/blob/main/packages/core/tests/config.test.ts#L60-L128.